yandextank/plugins/Android/plugin.py

```diff
@@ -4,7 +4,14 @@
 import urllib
 import sys
 import glob
+from multiprocessing import Process
 
+try:
+    import volta
+except Exception:
+    print "Please install volta. https://github.com/yandex-load/volta"
```

Comment:
You should reraise this exception here if you want to stop test execution. Reraise it with your message.